pages/bookings/index.tsx

```diff
@@ -224,7 +224,11 @@ export default function Bookings({ bookings }) {
                                               <Menu.Item>
                                                 {({ active }) => (
                                                   <a
-                                                    href={window.location.href + "/../cancel/" + booking.uid}
+                                                    href={
+                                                      process.env.NEXT_PUBLIC_APP_URL +
+                                                      "/../cancel/" +
+                                                      booking.uid
+                                                    }
```

Comment:
```suggestion
                                                    href={`${process.env.NEXT_PUBLIC_APP_URL}/cancel/${booking.uid}`}
```

Reply:
can u please open a new PR for these changes? 🙏